datafusion/core/src/physical_plan/joins/utils.rs

```diff
@@ -917,6 +917,23 @@ pub(crate) fn get_anti_indices(
         .collect::<UInt32Array>()
 }
 
+/// Get unmatched and deduplicated indices
+pub(crate) fn get_anti_u64_indices(
```

Comment:
Could we merge `get_anti_u64_indices/get_semi_u64_indices` into one function?
Because they are almost the same except `(!bitmap.get_bit(idx))`

Reply:
Yes, we can merge these two.

Further more, `get_semi_indices` and `get_semi_u64_indices`(anti) can also be merged to one function with generic.

I think it is relative to comment https://github.com/apache/arrow-datafusion/pull/5156#discussion_r1100068420, and @liukun4515 plans to refactor them.